proto/umee/auction/v1/query.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package umee.auction.v1;
+
+import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
+import "gogoproto/gogo.proto";
+import "umee/auction/v1/auction.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/umee-network/umee/v6/x/auction";
+
+// Query defines the gRPC querier service.
+service Query {
+  // QueryRewardParams queries parameters for the reward auciton.
+  rpc RewardParams(QueryRewardParams) returns (QueryRewardParamsResponse) {
+    option (google.api.http).get = "/umee/auction/v1/rewards/params";
+  }
+  // RewardAuction queries the information of the auction by ID. If ID is ommitted, returns
+  // current reward auction params.
+  rpc RewardAuction(QueryRewardAuction) returns (QueryRewardAuctionResponse) {
+    option (google.api.http).get = "/umee/auction/v1/rewards/{id}";
```

Comment:
```suggestion
    option (google.api.http).get = "/umee/auction/v1/rewards";
```
here `id ` should be query param because we already have `"/umee/auction/v1/rewards/params";` endpoint